JavaScript/JavaScriptSDK/AppInsights.ts

```diff
@@ -130,6 +133,8 @@ module Microsoft.ApplicationInsights {
                 (pageName, pageUrl, pageVisitTime) => this.trackPageVisitTime(pageName, pageUrl, pageVisitTime));
 
             if (!this.config.disableAjaxTracking) { new Microsoft.ApplicationInsights.AjaxMonitor(this); }
+            
+            DataLossAnalyzer.reportLostItems();
```

Comment:
for itemQueued function we are checking in DA is enabled in the caller code and for reportLostItems we are checking inside the function. Do we care if this consistent?